CalibCalorimetry/HcalAlgos/src/HcalPulseShapes.cc

```diff
@@ -265,6 +268,278 @@ void HcalPulseShapes::computeHFShape() {
 }
 
 
+void HcalPulseShapes::computeSiPMShapeData2017()
+{
+  //From Jay Lawhorn: derived from data Edward Laird phase scan may2017
```

Comment:
hi @mariadalfonso - is there a run number or presentation associated with this phase scan?